01-algorithms/02-algorithms-divide-conquer.md

```diff
@@ -46,246 +44,163 @@ QuickSort is an algorithm which takes a divide-and-conquer approach to sorting a
+1. Base Case: If there's only one element in the list, update the min and max by doing two comparisons
+    1. If the min is bigger than the value of the one element in the list, update the min
+    1. If the max is smaller than the value of the one element in the list, update the max
+1. Recursively call the function on both halves of the list
+1. Return the min and max values
 
-How do we efficiently merge two arrays? An auxiliary array of size _n_ is often used to facilitate the merge steps, after which the values are copied from the auxiliary array back to the original array.
+In this algorithm we are dividing the list up until there is only one element, making comparisons of the minimum and maximum values to this one element and updating the values if needed, and then combining the results by storing the minimum and maximum values encountered during our recursion.
 
-### !end-callout
+![Performing a modified version of Merge Sort to identify the minimum and maximum values of an array](images/algorithmic-strategies_divide-and-conquer_min-max.png)  
+ _Fig. Performing a modified version of Merge Sort to identify the minimum and maximum values of an array._
 
-The two-way merging continues until there are no more subarrays and the original array is completely sorted.
+### !callout-info
 
-### Divide and Conquer Complexity in Merge Sort
+## Is there an echo in here?
 
-Since merge sort always divides the list in half at each divide step, there will be _log n_ levels in the division phase. By keeping data in place, and only adjusting indices to keep track of where the subarrays are located, at each level, there are at most _n_ calculations, for a time complexity of the division phase of _O(n log n)_.
+You may notice similarities between this algorithm and the Merge Sort algorithm described above. If so, that's great! The Merge Sort and Binary Search algorithms are excellent models for a large number of divide-and-conquer algorithms. Due to this fact, it is recommended to spend more time understanding Merge Sort and Binary Search than any other divide-and-conquer algorithms. 
 
-In the merge phase, there will still be _log n_ levels of mergers, and at each level of the merge, there are _n_ copies from the split arrays into the auxiliary array, and _n_ copies back into the original array, giving a total complexity for the merge phase of _O(2n log n)_.
+### !end-callout
 
-Combining the divide and merge phases, we get a total complexity of _O(3n log n)_, which after dropping the coefficient gives us _O(n log n)_.
+Spend *no more than 15 minutes* writing a solution to the problem detailed above.
 
-<br />
+<details style="max-width: 700px; margin: auto;">
 
-<details>
+<summary>Click here to see a sample solution</summary>
 
-<summary>We can view an example of merge sort making use of index tracking by expanding this section.</summary>
+```py
```

Comment:
💫 For whatever reason, I could not get code blocks within a details tab to register as code blocks so kudos to you for figuring that out